kolibri/plugins/coach/assets/src/views/home/HomePage/OverviewBlock.vue

```diff
@@ -120,7 +120,7 @@
         context: 'Refers to the learner or learners who are in a class.',
       },
       viewLearners: {
```

Comment:
Let's get rid of the strings in both places here and add a string to the coach common strings, called `viewLearnerDevices`.